lib/newProfile.ml

```diff
@@ -16,11 +16,27 @@ module MiniJson = struct
     | `List of t list
   ]
 
+  (* https://www.ietf.org/rfc/rfc4627.txt *)
+  let prstring ch s =
+    let buf = Buffer.create (String.length s) in
+    let encode c =
+      match c with
+      | '\b' -> Buffer.add_string buf "\\b"
+      | '\t' -> Buffer.add_string buf "\\t"
+      | '\n' -> Buffer.add_string buf "\\n"
+      | '\012' -> Buffer.add_string buf "\\f"
+      | '\r' -> Buffer.add_string buf "\\r"
+      | '\x00'..'\x21' | 0x7F ->
+          Buffer.add_string buf (Printf.sprintf "\\u%04x" (Char.code c))
+      | '"' -> Buffer.add_string buf "\\\""
+      | '\\' -> Buffer.add_string buf "\\\\"
+      | c -> Buffer.add_char buf c
```

Comment:
Do we want to go with the minimal quoting?
```suggestion
      | '\x00'..'\x1F' ->
          Buffer.add_string buf (Printf.sprintf "\\u%04x" (Char.code c))
      | '"' -> Buffer.add_string buf "\\\""
      | '\\' -> Buffer.add_string buf "\\\\"
      | c -> Buffer.add_char buf c
```

Reply:
Should work, tho I'd suggest to follow the yojson set for the reasons stated in the discussion.